packages/gcp_vertexai/data_stream/metrics/fields/fields.yml

```
@@ -75,6 +75,70 @@
         - name: prediction_latencies
           type: histogram
           description: Online prediction latency of the deployed model.
-- name: gcp.labels.resource.location
-  type: keyword
-  description: Location of the resource
\ No newline at end of file
+## Label Fields
+- name: gcp.labels
+  description: Google Vertex AI dimension fields.
```

```suggestion
  description: Google Vertex AI labels.
```

Lets not use the word dimensions. 
It can get confusing with TSDB dimension.